### PR TITLE
Bump version to v1.2.1

### PR DESCRIPTION
**Pull request by dongluochen**
Signed-off-by: Dong Chen <dongluo.chen@docker.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 1.2.1 (2016-04-22)
+## 1.2.1 (2016-05-03)
+
+#### Scheduler
+
+- Add containerslots filter to allow container number limit on a node
```
Comment (nishanttotla) [inline CHANGELOG.md]: Can we change this to `...to allow for limiting number of containers on a node`? It'll be more clear in my opinion.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,16 +14,27 @@
 - Return an error when assertion fails in hijack
 - Return an error when Image Pull fails
 - Fix image pull bug (wait until download finishes)
+- Fix and document some api response status code
```
Comment (nishanttotla) [inline CHANGELOG.md]: `code` -> `codes`

**Comment by nishanttotla** [comment]
LGTM

**Comment by abronan** [comment]
LGTM

**Comment by vieux** [comment]
:tada:

**Pull request merged**